package.json

```diff
@@ -7,8 +7,8 @@
   "scripts": {
     "clean": "turbo run clean --force --parallel",
     "build": "turbo run build",
-    "test": "turbo run test",
-    "test:integration": "yarn build-test-packages && turbo run test:integration",
```

Comment:
as with jsv3, turbo is a slow way to run tests because it spins up separate vite processes. 

A singular vitest process with a view of all tests is much faster for test compilation.